tests/baselines/reference/parserArrowFunctionExpression14(target=es3).errors.txt

```diff
@@ -0,0 +1,38 @@
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,1): error TS2304: Cannot find name 'a'.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,11): error TS8010: Type annotations can only be used in TypeScript files.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,20): error TS8009: The '?' modifier can only be used in TypeScript files.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,23): error TS8010: Type annotations can only be used in TypeScript files.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,32): error TS8010: Type annotations can only be used in TypeScript files.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,40): error TS2304: Cannot find name 'd'.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js(1,46): error TS2304: Cannot find name 'e'.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileTs.ts(1,1): error TS2304: Cannot find name 'a'.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileTs.ts(1,40): error TS2304: Cannot find name 'd'.
+tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileTs.ts(1,46): error TS2304: Cannot find name 'e'.
+
+
+==== tests/cases/conformance/parser/ecmascript5/ArrowFunctionExpressions/fileJs.js (7 errors) ====
+    a() ? (b: number, c?: string): void => d() : e; // Not legal JS; "Unexpected token ':'" at first colon

```

Comment:
Is this comment supposed to reflect the runtime error?

Reply:
Yes, that's the parser error I get from the browser. I can drop these if they are confusing, I only had them in there to convince myself that in every case, the only ones with TS-y meanings were ones that were not valid JavaScript.